### DISCUSSION TITLE
Using FairIon for neutron clusters (z=0)

### DISCUSSION THREAD

**Discussion by himyss** [FairRootGroup/FairRoot#1539] [Q&A]
Dear FAIRROOT community, I am wondering if it is correct to use FairIon class for neutron clusters with Z=0?

**Comment by karabowi** [comment]
Hi Ivan, do you know if neutron clusters are supported by the transport engine, i.e. Geant3 or Geant4?

**Reply by himyss** [reply]
I am not using FairIon in order to transfer 4n system, but to define the excitation energy for it and simulate its decay

**Reply by fuhlig1** [reply]
Hi Ivan,

the issue is that FairRoot doesn't do any decays or transport on it's own but uses either Geant3 or Geant4 for this tasks. So if the dedicates MC transport models don't support the decay and transport of the 4n systems FairRoot can't handle that type of particles.
